src/wp-admin/includes/post.php

```diff
@@ -1392,6 +1392,7 @@ function get_sample_permalink( $id, $title = null, $name = null ) {
 	$original_status = $post->post_status;
 	$original_date   = $post->post_date;
 	$original_name   = $post->post_name;
+	$original_filter = $post->post_filter;
```

Comment:
The name of the property is `filter` :relaxed:

Reply:
> The name of the property is `filter` ☺️

Excellent point 🙈 
* 09ea43f2b6369cb23c2327c3cd649cdc4f9b2cd2 uses the correct property name
* 74aeb8094f5cf69c71cdf31d2f68f93aa6020990 adds a test to ensure the function call does not change the property value

Reply:
Further follow up, I hadn't seen the PR against this branch until after my comment above.

I've merged your test in and modified it slightly in 057b52b76ef37f47ce706bbd6157c8370415d2bc to compare the properties both within the filter and after the function runs.